docs/sdk/gaiacli.md

```diff
@@ -7,18 +7,20 @@
 There are three types of key representations that are used:
 
 - `cosmosaccaddr`
-  * Derived from account keys generated by `gaiacli keys add`
-  * Used to receive funds
-  * e.g. `cosmosaccaddr15h6vd5f0wqps26zjlwrc6chah08ryu4hzzdwhc`
+
+  - Derived from account keys generated by `gaiacli keys add`
+  - Used to receive funds
+  - e.g. `cosmosaccaddr15h6vd5f0wqps26zjlwrc6chah08ryu4hzzdwhc`
 
 - `cosmosaccpub`
-  * Derived from account keys generated by `gaiacli keys add`
-  * e.g. `cosmosaccpub1zcjduc3q7fu03jnlu2xpl75s2nkt7krm6grh4cc5aqth73v0zwmea25wj2hsqhlqzm`
+
+  - Derived from account keys generated by `gaiacli keys add`
+  - e.g. `cosmosaccpub1zcjduc3q7fu03jnlu2xpl75s2nkt7krm6grh4cc5aqth73v0zwmea25wj2hsqhlqzm`
 
 - `cosmosvalpub`
-  * Generated when the node is created with `gaiad init`.
-  * Get this value with `gaiad tendermint show_validator`
-  * e.g. `cosmosvalpub1zcjduc3qcyj09qc03elte23zwshdx92jm6ce88fgc90rtqhjx8v0608qh5ssp0w94c`
+  - Generated when the node is created with `gaiad init`.
+  - Get this value with `gaiad tendermint show_validator`
+  - e.g. `cosmosvalpub1zcjduc3qcyj09qc03elte23zwshdx92jm6ce88fgc90rtqhjx8v0608qh5ssp0w94c`
```

Comment:
++

Reply:
@zramsay please include a linter for markdown